### DISCUSSION TITLE
Wanna say thank you

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#1672] [General]
Hi,

I accidentally came across this website during my search for information on Proxmox. I was pleasantly surprised by these scripts! I used Docker on unRAID but I'm moving to Proxmox. Also I'm going to change my NAS software for TrueNAS or to the NAS OS of a newcomer in the NAS space. I appreciate the hard work people put in these scripts. Many thank you's!!!

**Comment by elminster13** [comment]
Second that! and I bought you a coffee or four too...

**Comment by ghost** [comment]
Yeah, I did the same seems like the right thing to do!

**Comment by Drallas** [comment]
Also, BiG thanks from me. 

I had this project bookmarked 2 years+ ago. 

I thought it was only an optimization script., but checked it again yesterday and now there is a wonderful collection of LXC scripts!

PS: Is it also possible to support via GitHub Sponsoring?
